BASE/src/ServerTelemetryChannel/Implementation/TransmissionPolicy/TransmissionPolicyCollection.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation.TransmissionPolicy
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    internal class TransmissionPolicyCollection : IDisposable
+    {
+        private readonly IEnumerable<TransmissionPolicy> policies;
+        private bool isDisposed;
+
+        public TransmissionPolicyCollection(INetwork network, IApplicationLifecycle applicationLifecycle)
+        {
+            this.policies = new TransmissionPolicy[]
+            { 
+#if NETFRAMEWORK
+                // We don't have implementation for IApplicationLifecycle for .NET Core
+                new ApplicationLifecycleTransmissionPolicy(applicationLifecycle),
+#endif
+                new ThrottlingTransmissionPolicy(),
+                new ErrorHandlingTransmissionPolicy(),
+                new PartialSuccessTransmissionPolicy(),
+                new NetworkAvailabilityTransmissionPolicy(network),
+            };
+        }
+
+        /// <summary>
+        /// Constructor for unit tests only.
```

Comment:
This comment seems to be incorrect. `Default` calls this constructor and referred in `Transmitter`.